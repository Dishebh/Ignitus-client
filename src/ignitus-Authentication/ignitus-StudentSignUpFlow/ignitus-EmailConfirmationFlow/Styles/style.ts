import styled from '@emotion/styled';
import { LazyImage } from '../../../../ignitus-Shared/ignitus-DesignSystem/ignitus-Assets/Ignitus-LazyImage/Component';

// export const NewsletterImage = styled.img``;
export const NewsletterImage = styled(LazyImage);

export const BottomSection = styled.div`
  margin-top: 3rem;
`;

export const TopMiddleSection = styled.div`
  margin-top: 1rem;
`;
