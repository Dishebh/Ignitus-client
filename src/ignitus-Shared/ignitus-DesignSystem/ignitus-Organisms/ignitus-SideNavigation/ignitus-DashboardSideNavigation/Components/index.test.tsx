import React from 'react';
import { mount, render } from 'enzyme';

import DashboardNavigation, { Layer } from './index';
import { Paragraph } from '../../../../ignitus-Atoms/typography';
import { SideNavIcon } from '../style';

let wrapper;
beforeEach(() => {
  wrapper = mount(<DashboardNavigation />);
});

describe('<DefaultButtonWithIcon />', () => {
  it('should rendered', () => {
    const tree = render(<DashboardNavigation />);
    expect(tree).toMatchSnapshot();
  });

  it('should rendered all paragraphs', () => {
    expect(wrapper.find(Paragraph).length).toEqual(6);
  });

  it('should rendered all icons', () => {
    expect(wrapper.find(SideNavIcon).length).toEqual(6);
  });

  it('should rendered all layers', () => {
    expect(wrapper.find(Layer).length).toEqual(6);
  });
});
