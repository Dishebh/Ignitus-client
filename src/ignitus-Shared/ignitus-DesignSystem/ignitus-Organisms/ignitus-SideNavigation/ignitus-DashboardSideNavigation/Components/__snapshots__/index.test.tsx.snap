// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DefaultButtonWithIcon /> should rendered 1`] = `
<div
  class="css-17gamlc"
>
  <div
    class="css-1l6rv91"
  >
    <svg
      aria-hidden="true"
      aria-label="SaveIcon"
      class="css-1rj63zu"
      data-icon="saveIcon"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title>
        SaveIcon
      </title>
      <path
        d="M19 12v7H5v-7H3v7c0 1.1.9 2 2 2h14a2 2 0 002-2v-7h-2zm-6 .67l2.59-2.58L17 11.5l-5 5-5-5 1.41-1.41L11 12.67V3h2z"
      />
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
    </svg>
    <p
      class="css-7f2qlu"
      color="#000066"
    >
      Saved
    </p>
  </div>
  <div
    class="css-14comxu"
  >
    <svg
      aria-hidden="true"
      aria-label="HeartIcon"
      class="css-1rj63zu"
      data-icon="networkIcon"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M13.35 20.13a2 2 0 01-2.69-.01l-.11-.1C5.3 15.27 1.87 12.16 2 8.28c.06-1.7.93-3.33 2.34-4.29A5.8 5.8 0 0112 5.09a5.78 5.78 0 017.66-1.1A5.46 5.46 0 0122 8.28c.14 3.88-3.3 6.99-8.55 11.76l-.1.09z"
      />
    </svg>
    <p
      class="css-7f2qlu"
      color="#000066"
    >
      Liked
    </p>
  </div>
  <div
    class="css-14comxu"
  >
    <svg
      aria-label="Apply Icon"
      class="css-1rj63zu"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title>
        Apply Icon
      </title>
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M18 19H6c-.55 0-1-.45-1-1V6c0-.55.45-1 1-1h5c.55 0 1-.45 1-1s-.45-1-1-1H5c-1.11 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2v-6c0-.55-.45-1-1-1s-1 .45-1 1v5c0 .55-.45 1-1 1zM14 4c0 .55.45 1 1 1h2.59l-9.13 9.13c-.39.39-.39 1.02 0 1.41.39.39 1.02.39 1.41 0L19 6.41V9c0 .55.45 1 1 1s1-.45 1-1V3h-6c-.55 0-1 .45-1 1z"
      />
    </svg>
    <p
      class="css-7f2qlu"
      color="#000066"
    >
      Applied
    </p>
  </div>
  <div
    class="css-14comxu"
  >
    <svg
      aria-hidden="true"
      aria-label="NetworkIcon"
      class="css-1rj63zu"
      data-icon="networkIcon"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title>
        NetworkIcon
      </title>
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
      <path
        d="M8 10H5V7H3v3H0v2h3v3h2v-3h3v-2zm10 1c1.66 0 2.99-1.34 2.99-3a2.99 2.99 0 00-3.9-2.86 4.93 4.93 0 010 5.72c.28.09.59.14.91.14zm-5 0c1.66 0 2.99-1.34 2.99-3S14.66 5 13 5a3 3 0 100 6zm6.62 2.16A3.7 3.7 0 0121 16v2h3v-2c0-1.54-2.37-2.49-4.38-2.84zM13 13c-2 0-6 1-6 3v2h12v-2c0-2-4-3-6-3z"
      />
    </svg>
    <p
      class="css-7f2qlu"
      color="#000066"
    >
      Network
    </p>
  </div>
  <div
    class="css-14comxu"
  >
    <svg
      aria-hidden="true"
      aria-label="MentorsIcon"
      class="css-1rj63zu"
      data-icon="mentorsIcon"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title>
        MentorsIcon
      </title>
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
      <path
        d="M16 11c1.66 0 2.99-1.34 2.99-3S17.66 5 16 5a3 3 0 100 6zm-8 0c1.66 0 2.99-1.34 2.99-3S9.66 5 8 5a3 3 0 100 6zm0 2c-2.33 0-7 1.17-7 3.5V19h14v-2.5c0-2.33-4.67-3.5-7-3.5zm8 0c-.29 0-.62.02-.97.05 1.16.84 1.97 1.97 1.97 3.45V19h6v-2.5c0-2.33-4.67-3.5-7-3.5z"
      />
    </svg>
    <p
      class="css-7f2qlu"
      color="#000066"
    >
      Mentors
    </p>
  </div>
  <div
    class="css-jf3c8w"
  >
    <svg
      aria-hidden="true"
      aria-label="MessageIcon"
      class="css-1rj63zu"
      data-icon="messageIcon"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title>
        MessageIcon
      </title>
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
      <path
        d="M20 4H4a2 2 0 00-1.99 2L2 18c0 1.1.9 2 2 2h16a2 2 0 002-2V6a2 2 0 00-2-2zm0 14H4V8l8 5 8-5v10zm-8-7L4 6h16l-8 5z"
      />
    </svg>
    <p
      class="css-7f2qlu"
      color="#000066"
    >
      Message
    </p>
  </div>
</div>
`;
